### PR TITLE
Declutter Robot Loop

### PR DESCRIPTION
**Pull request by TrueDemoknight**
Removed old periodic method calls from the robot loop.

### CODE REVIEW

File: src/main/java/frc/robot/Robot.java [hunk 1]
```diff
@@ -319,38 +319,6 @@ public void RobotLoop() {
...
-
-    // Operator Controls
-    if (mOperatorInterface.getTurretAdjustLeft()) {
-      // manual turret aim
-    } else if (mOperatorInterface.getTurretAdjustRight()) {
-      // manual turret aim
-    }
-
-    // Camera Swap
-    if (mOperatorInterface.getCameraSwap()) {
```
Comment (jhinspeter) [inline src/main/java/frc/robot/Robot.java:340]: Camera Swap can stay

File: src/main/java/frc/robot/Robot.java [hunk 2]
```diff
@@ -319,38 +319,6 @@ public void RobotLoop() {
...
-      mBallIndicator.checkTimer();
-    }
-
-    // Climb
-    if (mOperatorInterface.getClimb()) {
-      // climb!
-    }
-
-    // Operator Controls
-    if (mOperatorInterface.getTurretAdjustLeft()) {
```
Comment (jhinspeter) [inline src/main/java/frc/robot/Robot.java:333]: Manual Turret adjustment (Left and Right) can stay

**Review by jhinspeter — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/Robot.java (hunk 2 above)
Comment (TrueDemoknight) [reply src/main/java/frc/robot/Robot.java]: Fixed

Comment (TrueDemoknight) [reply src/main/java/frc/robot/Robot.java:333]: Fixed

File: src/main/java/frc/robot/Robot.java (hunk 1 above)
Comment (TrueDemoknight) [reply src/main/java/frc/robot/Robot.java:340]: Fixed

Comment (TrueDemoknight) [reply src/main/java/frc/robot/Robot.java:340]: Fixed

File: src/main/java/frc/robot/Robot.java (hunk 2 above)
Comment (TrueDemoknight) [reply src/main/java/frc/robot/Robot.java:333]: Fixed

**Review by jhinspeter — APPROVED** [review]

**Pull request merged**